src/System.Composition/tests/ConstraintTests.cs

```diff
@@ -50,5 +65,30 @@ public void GenericPartDiscoveryIncludesAPartWhenItsConstraintOnTypeParameterIsA
             Assert.Contains<Type>(typeof(ThingHandler<IBook>), handlerTypes);
             Assert.Contains<Type>(typeof(BookHandler<IBook>), handlerTypes);
         }
+
+        [Fact]
+        [ActiveIssue(24903, TargetFrameworkMonikers.NetFramework)]
```

Comment:
https://github.com/dotnet/corefx/issues/24903 is closed is this the right issue?